packages/apps/job-launcher/server/src/modules/storage/storage.service.ts

```diff
@@ -54,9 +53,9 @@ export class StorageService {
       return Buffer.from(data);
     } catch (error) {
       if (error.response?.status === HttpStatus.NOT_FOUND) {
-        throw new FileNotFoundError(url);
+        throw new NotFoundError(`${ErrorStorage.NotFound}: ${url}`);
```

Comment:
It's confusing to have `NotFoundError` here because it might end up to 404, when it's not, it's about processing error, which is more about 422 or 500